test/e2e/lib/framework/pageSetups.ts

```diff
@@ -15,6 +15,7 @@ export interface SetupOptions {
 export type SetupFactory = (options: SetupOptions, servers: Servers) => string
 
 const isBrowserStack =
+  'services' in browser.config &&
   browser.config.services &&
```

Comment:
Is it required to have both checks?

Reply:
Yes, because `browser.config` is something like `{ services: something } | { other stuff }`, so we need the `'services' in browser.config` otherwise TS yield as us because "services" is not in `{ other stuff }`.